usaspending_api/settings.py

```diff
@@ -61,6 +60,10 @@
 # Define local flag to affect location of downloads
 IS_LOCAL = True
 
+# Indicates which environment is sending traces to Grafana.
+# This will be overwritten by Ansible
+TRACE_ENV = "unspecified"
```

Comment:
Do we not want to define this in our different config files?

Reply:
We tried this but found that those config files were not actually being used outside of management commands. Otherwise, we could have just used the existing `ENV_CODE` variable. Double checking with @boozallendanny that we're also overwriting this outside of deploys (places like management commands)

Reply:
Checked on this with Danny. Looks like another change is needed on the infra side to add the override to the bulk downloads. No tracing is taking place in other management commands.